Foundation/Date.swift

```diff
@@ -275,3 +275,12 @@ extension Date : Codable {
         try container.encode(self.timeIntervalSinceReferenceDate)
     }
 }
+
+internal extension FILETIME {
+  init(from date: Date) {
+    let UNIXTime: Int64 =
+        Int64((date.timeIntervalSince1970 * 10000000) + 116444736000000000)
+    self = FILETIME(dwLowDateTime: DWORD(UNIXTime & 0xffffffff),
```

Comment:
`self = FILETIME(from: date.timeIntervalSince1970)` should work here

Reply:
Hmm, I think I just prefer to use that rather than add the extension!  Great idea!